postgres/pg-stack.yml

```
@@ -3,7 +3,7 @@ version: '3.1'
 
 services:
   db:
-    image: postgres:15-alpine
+    image: postgres:alpine
```

```suggestion
    image: postgres:16-alpine
```

Keep the PRs reproducible